srcpkgs/osmin/template

```diff
@@ -0,0 +1,30 @@
+# Template file for 'osmin'
+pkgname=osmin
+version=1.8.7
+revision=1
+build_style=cmake
+configure_args="-DCMAKE_C_COMPILER=/usr/bin/clang -DCMAKE_CXX_COMPILER=/usr/bin/clang++ ."
+hostmakedepends="qt5-qmake pkg-config qt5-host-tools git"
+makedepends="qt5-declarative-devel qt5-3d clang libgomp-devel
+	libomp-devel qt5-declarative-devel qt5-tools-devel
+	qt5-svg-devel   qt5-quickcontrols2-devel qt5-sensors-devel
+	qt5-multimedia-devel qt5-remoteobjects-devel qt5-location-devel"
+short_desc="GPS Navigator On-Road/Off-Road"
+
+archs="~armv6l-musl ~armv7l ~i686" # libomp-devel
```

Comment:
this would be better done as marking them broken:
```sh
case "$XBPS_TARGET_MACHINE" in
	x86_64*|aarch64*|ppc64*) ;;
	*) broken="no libomp"
esac
```